code/controllers/Processes/air.dm

```diff
@@ -107,6 +127,7 @@ var/global/datum/controller/process/air_system/air_master
 			return
```

Comment:
Hmm, yes, it'll skip the scheck for a single excited group, but will go back to schecking after the next thing it processes. There's no way skipping one scheck would throw the timing off, but I probably should've put one there anyway.